JetNews/build.gradle

```diff
@@ -14,18 +14,18 @@
  * limitations under the License.
  */
 
-buildscript {
-    ext.kotlin_version = '1.4.0'
-    ext.compose_version = '1.0.0-alpha01'
+import com.example.jetnews.buildsrc.Libs
+import com.example.jetnews.buildsrc.Versions
 
+buildscript {
     repositories {
         google()
         jcenter()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.2.0-alpha08'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath Libs.androidGradlePlugin
```

Comment:
JetNews doesn't use buildSrc on purpose. We decided to leave JetNews copypasteable.